packages/gestalt-codemods/123.0.0/button-role-link-deprecation.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
can we add tests for this?

Reply:
should be easy, you did the hard part hehe

Reply:
DONE